studentClass/src/studentclass/StudentClass.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package studentclass;
+
+/**
+ *
+ * @author DEE
+ */
+public class StudentClass {
+
+    /**
+     * @param args the command line arguments
+     */
+    static class Student {
+
+    private String name;
+    private String address;
+    private int age;
+    private String phoneNumber;
+
+    public void setName(String n) {
+        name = n;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setAddress(String location) {
+        address = location;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAge(int years) {
+        age = years;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setPhoneNumber(String PhoneNum) {
+        phoneNumber = PhoneNum;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+}
+    public static void main(String[] args) {

```

Comment:
Sorry. I thought you had two student classes. You could have put the `main` method in another class, e.g., in a `StudentDatabase` class. Putting a `main` method here is okay but in more realistic applications you would normally not do so as there are probably many classes.